package.json

```diff
@@ -35,7 +35,7 @@
     "@bdelab/roav-mep": "1.2.0",
     "@bdelab/roav-ran": "1.0.33",
     "@dotenvx/dotenvx": "^1.24.4",
-    "@levante-framework/core-tasks": "1.0.0-beta.39",
+    "@bdelab/roar-levante-tasks": "0.0.1",
```

Comment:
Can you bump this version now that I've approved/merged the PR in the roar-levante-tasks repo?